src/entities/college.entity.ts

```diff
@@ -0,0 +1,45 @@
+import { 
+  Entity, 
+  PrimaryGeneratedColumn, 
+  Column, 
+  CreateDateColumn, 
+  UpdateDateColumn, 
+  DeleteDateColumn, } from "typeorm";
+
+import { v4 as uuidv4 } from 'uuid';
+
+@Entity()
+export class College {
+  @PrimaryGeneratedColumn("uuid")
+  id!: string;
+
+  @Column({ type: "varchar", length: 50, nullable: false })
+  name!: string;
+
+  @Column({ type: "varchar", nullable: true })
```

Comment:
what you have in your sql code is not null, so why is it nullable in your entity (plus the ?)

Reply:
again, are you saying every college does not have its own uniform